source/upgrade/notify-admin.rst

```diff
@@ -0,0 +1,25 @@
+Notify admin
+============
+
+Some Mattermost features are limited to specific plans. Users who want to access these unavailable features can request access through their System Admin by sending a notification.
+
+If access to a feature requires a plan upgrade, System Admins receive notifications of these requests in order to collect data before upgrading. For example, if only one end-user requests access to AD/LDAP, it probably isn't necessary to upgrade. However, if 20 users, including Team and Channel Admins request it, you may want to consider upgrading to support this need.
+
+This feature is designed for informational purposes only, and no action is required unless you want to take action. Your Mattermost instance's functionality is not affected if you choose not to upgrade.
+
+Notifications
+-------------
+
+Notifications are triggered by users. The very first time a user sends a request to upgrade or start a trial, a bot message is sent to all System Admins indicating the feature or functionality the user has requested that requires an upgrade or trial. This bot message is listed in the **Direct Messages** section of the channel sidebar.
+
+Any requests that happen after this first request are sent 14 days later, in a summary format.
```

Comment:
Propose changing this line to:

Subsequent request notifications are received by admins at most every 14 days. When a notification is received by an admin, a 14-day cool-off period ensues. Any request by end user(s) in the middle of the cool-off period will be held off until 14 days later in a summarized format.